taxi_service/settings.py

```diff
@@ -26,7 +26,7 @@
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = True
 
-ALLOWED_HOSTS = []
+ALLOWED_HOSTS = ["127.0.0.1", "localhost"]
```

Comment:
The `ALLOWED_HOSTS` setting is currently configured for local development. Ensure to update this with your domain names when deploying to production.